cartesi-rollups/challenges/overview.md

```diff
@@ -0,0 +1,84 @@
+
+### Challenge
+
+In contrast to the majority of proposed Layer 2 rollup solutions, which primarily focus on enhancing transaction throughput and minimizing fees (essentially facilitating more transactions at a lower cost), the current **v1.0** release is dedicated to expanding the *computational capacity* (as opposed to *transaction scalability* alone) to allow the creation of decentralized applications that were simply not possible to build before.
+
+### Solution strategy
+
+A potential path that can be considered to achieve transaction scalability is the introduction of an *Aggregator*, often associated with a Sequencer. This component might enhance the overall efficiency and capacity of the rollup solution. Moreover, an exciting area for exploration is deploying Cartesi on top of other Layer-2 solutions like Optimism and Arbitrum, potentially expanding the options for enhanced transaction scalability.
+
+## Disputes
+
+### Challenge
+
+The **1.0** release lacks support for dispute resolution.
+
+### Solution strategy
+
+From its inception, the Cartesi framework has the potential to integrate dispute resolution features. Cartesi's community-driven iterations have leveraged dispute resolution for [Compute](/compute/overview), which may be interesting to investigate further after other features and improvements, like the Microarchitecture enhancement and recent research on NxN dispute resolution, have been fully addressed. 
+
+## Large data as input
+
+### Challenge
+
+The current Rollups framework does not support very large data inputs (e.g., from IPFS), limiting DApps based on Cartesi Rollups to rely solely on on-chain data.
+
+### Solution strategy
+
+A promising research area is enhancing Cartesi Rollups' data capabilities. Potential advancements in the Ethereum ecosystem, like [EIP-4844](https://www.eip4844.com/) (proto-danksharding), might make on-chain data more affordable. Moreover, layering Cartesi on top of other rollup solutions can offer data advantages.
+
+Another feature for community consideration could be a mechanism assisting front-end clients in dividing large pieces of data into smaller chunks and reassembling them within the back-end. This approach would be similar to the [Logger](/compute/logger_drive/) feature used in Cartesi Compute.
+
+Additionally, potential future exploration might focus on integrating Rollups with Data Availability solutions like zkPorter or using a Data Availability Committee solution for quorum-based validator sets.
+
+## Composability
+
+### Challenge
+
+The current **v1.0** release lacks composability, which is the ability of multiple DApps to directly interact with each other, for example by having one application directly send inputs to another one without going through the base layer. This would enable seamless communication and data exchange between DApps. The most important use case for this feature would be to allow users to easily and cheaply transfer assets from one DApp to another. 
```

Comment:
I wouldn't say "lacks composability"! Cartesi Rollups do have composability in the form of vouchers, so it is as generic as possible. The problem is in the price (its an L1 tx) and the delay (has to wait for the challenge period). 

I'd say something like: "The current **v1.0** release doesnt offer great composability" or "The current release still faces a lot of composability hurdles". Something that doesnt imply the inexistence of composability